src/OidcClient.test.ts

```diff
@@ -371,6 +373,10 @@ describe("OidcClient", () => {
             const response = await subject.useRefreshToken({ state });
 
             // assert
+            expect(exchangeRefreshTokenMock).toHaveBeenCalledWith( {
+                refresh_token: "refresh_token",
+                scope: "openid",
+            });
```

Comment:
added an assert to an existing unit test, let me know if you'd rather have this tested differently.